src/Lessons/Task9.java

```diff
@@ -0,0 +1,13 @@
+package Lessons;
+
+public class Task9 {
+    public static void main(String[] args) {
+
+        int a = 21;
```

Comment:
Don't need to create separate variable, just set the direct number to the loop condition